site/docs/v1/tech/apis/index.adoc

```diff
@@ -117,6 +118,111 @@ Each API may document specific status codes and provide a specific reason for re
 |A lambda invocation failed during this API request. An event log will have been created with details of the exception. See [breadcrumb]#System -> Event Log#.
 |===
 
+== The PATCH HTTP Method
+
+There are three options for using `PATCH` operations. You choose between them by specifying a particular `Content-Type` on the request.
+
+.PATCH options
+[cols="1,1,1,1,1,1"]
+|===
+| Name | Content-Type | Array Behavior | Available Since | RFC Link | Client Library Support
+
+|FusionAuth default
+|`application/json`
+|Appends new values to an array. Deletion requires `GET` then `PUT`.
+|1.12.0
+|N/A
+|Yes
+
+|JSON Patch
+|`application/json-patch+json`
+|Uses operations to specify JSON modifications.
+|1.39.0
+|https://www.rfc-editor.org/rfc/rfc6902[RFC 6902]
+|No
+
+|JSON Merge Patch
+|`application/merge-patch+json`
+|If an array is provided, overwrite the existing array.
+|1.39.0
+|https://www.rfc-editor.org/rfc/rfc7396[RFC 7396]
+|No
+
+|===
+
+=== PATCH Example
+
+Here's an example of how the different options work when used to modify the roles of a link:/docs/v1/tech/apis/groups[Group] which has roles of "ceo" and "dev" to remove the "dev" role.
+
+If you are only modifying specific object fields, all three `PATCH` methods are equivalent. 
+
+==== FusionAuth Default
+
+With the default method, the correct way to remove the "dev" role is to request the group JSON, find the "ceo" role from the `roleIds` array, and use `PUT` to update the group object. 
+
+[source,shell,title=Updating an object's array with the default PATCH behavior]
+----
+API_KEY=...
+GROUP_ID=...
+APPLICATION_ID=...
+group_json=`curl -H 'Authorization: '$API_KEY \
+     -X GET \
+     https://local.fusionauth.io/api/group/$GROUP_ID`
+
+# construct new object without "dev" role
+ceoRoleId=`echo $group_json | jq '.group.roles."'$APPLICATION_ID'"[] | select(.name == "ceo")|.id'`
+name=`echo $group_json | jq '.group.name'`
+data=`echo $group_json | jq '.group.data'`
+new_group_json='{"group": {"name" : '$name', "data":'$data'}, "roleIds": ['$ceoRoleId']}'
+
+echo $new_group_json
+# PUT the modified object
+curl -H 'Authorization: '$API_KEY \
+     -X PUT \
+     -H 'Content-type: application/json' \
+     https://local.fusionauth.io/api/group/$GROUP_ID \
+     -d "$new_group_json"
+----
+
+==== JSON Patch
+
+With JSON Patch, you have a flexible set of operations that can update, remove or move fields in a JSON object. Please review https://www.rfc-editor.org/rfc/rfc6902[RFC 6902] for the full list of operations.
+
+[source,shell,title=Updating an object's array with JSON Patch]
+----
+API_KEY=...
+GROUP_ID=...
+ROLE_ID=...
```

Comment:
Could you use a more descriptive name for this variable? I had to go back a few times to be sure what the example was trying to achieve. If someone (like me) is not very familiar with these operations, it may not be clear whether `ROLE_ID` is the `ceo` role we are trying to keep, or the `dev` role we want to remove.

Reply:
Removed these entirely and just went with the json.